language-server/modules/langserver-core/src/test/resources/completion/annotation_ctx/source/moduleLevelAnnotation1.bal

```diff
@@ -1,3 +1,14 @@
-import ballerina/module1;
+public type AnnotationType record {
```

Comment:
Please check if this is related to this IDL client feature. I dont think do. https://github.com/ballerina-platform/ballerina-lang/pull/31342

Reply:
Updated